launch/launch/actions/execute_process.py

```diff
@@ -577,6 +583,16 @@ def __expand_substitutions(self, context):
             self.__logger.info("process details: cmd=[{}], cwd='{}', custom_env?={}".format(
                 ', '.join(cmd), cwd, 'True' if env is not None else 'False'
             ))
+        try:
+            emulate_tty = yaml.safe_load(
+                context.launch_configurations['emulate_tty']
+            )
```

Comment:
@wjwwood meta: I think `yaml` parsing here is fine for now, but I also think we need to get this straight in `launch`. We're already applying YAML parsing under the hood for parameters, maybe we should standardize it (or not)?

Reply:
In the original pull request I was going to mention this, but I didn't just to keep things moving. We have a function for this kind of parsing which we use in the condition evaluations:

https://github.com/ros2/launch/blob/master/launch/launch/conditions/evaluate_condition_expression_impl.py

It's a subset of YAML. I think we can/should stabilize on something. I don't mind using yaml, though we would need to depend on pyyaml. Which brings up another issue which is that launch doesn't declare a dependency on yaml after this pr. So that's not right. I think this is why I made the evaluation function in the first place. I think I will reuse that and drop the use of YAML for now. Allowing `true`(any cap)/`1`/`false`/`0` is documented in other parts of the code too.

Reply:
See 1b84e42